kfaraz/druid
# Cleanup TaskQueue

Pending:
- Use correct data structure for `tasks`

Tests
- Verify that the following are idempotent
  - add
  - shutdown
  - removeTaskInternal
- Add some tests for update order
